File: Validation_I/Validation_I/Model/StudentModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.ComponentModel.DataAnnotations;
namespace Validation_I
{
    public class StudentModel
    {
        [Key]
        public Guid StudentId { get; set; }

        [MyRequire(ErrorMessage="Ten sinh vien khong duoc trong")]
        
        public string StudentName { get; set; }
        public string StudentInformation { get; set; }

        [MyRequire(ErrorMessage="Nam hoc khong duoc de trong")]
        [MyMinLength(5,ErrorMessage="Khong duoc nho hon 5")]
        [MyMaxLength(100, ErrorMessage = "Nho hon 100")]
        public int StudentGrade { get; set; }
    }
}

File: Validation_I/Validation_I/StudentModel.js
    var StudentModel = function (StudentId, StudentName, StudentInformation, StudentGrade){
var self=this;
self.StudentId = ko.observable(StudentId).extend({});
self.StudentName = ko.observable(StudentName).extend({required: {params: False,message: 'Ten sinh vien khong duoc trong'},});
self.StudentInformation = ko.observable(StudentInformation).extend({});
self.StudentGrade = ko.observable(StudentGrade).extend({min: {params: 5,message: 'Khong duoc nho hon 5'},max: {params: 100,message: 'Nho hon 100'},required: {params: False,message: 'Nam hoc khong duoc de trong'},});
self.IsCreate = ko.observable(false);
                            self.IsEdit = ko.observable(false);
                            self.IsVisible = ko.observable(true);
                            self.OldValue = ko.observable({});

                            // Validate.
                            this.Errors = ko.validation.group(this);
                            this.isValid = ko.computed(function () {
                                return self.Errors().length == 0;
                            });
}